libyui-ncurses/CMakeLists.txt

```diff
@@ -24,6 +24,7 @@ project( libyui-ncurses )
 
 option( BUILD_SRC         "Build in src/ subdirectory"                on )
 option( BUILD_DOC         "Build class documentation"                 off )
+option( BUILD_PKGCONFIG   "Build pkg-config support files"            on  )
```

Comment:
IMHO we should always build them, unconditionally.
They are small and quick enough, including the pkg-config dependency.

Reply:
As discussed on IRC: We have a separate BUILD_XXX option for each subdirectory. This is consistent and simple, and it doesn't do implicit things.